README.md

```diff
@@ -1,44 +1,6 @@
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+## React Azure Maps Playground
+  - How to link local package version:
+    - run yarn watch in `azure-maps-react`
+    - run yarn link `azure-maps-react`
+    - go to the `azure-maps-playground` and run `yarn link "react-azure-maps"`
```

Comment:
typos, change azure-maps-react to react-azure-maps

Reply:
readme file will. be completely changed after major features will be done